### PR TITLE
read the rssi from the BLE sensors 

### PR DESCRIPTION
**Pull request by d0m1n1qu3**
## Description:

sometimes it is hard to find out why a sensor sends no data
with the rssi you got a value of the BLE signal strenght
and can optimize the location of the sensor or the ESP32

if the rssi got worse over time so the battery could also be the problem

rssi monitoring is a must have feature for radio signals :-)

again ESP32 only :)

## Checklist:
  - [x] The pull request is done against the latest dev branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on core ESP8266 V.2.7.2
  - [x] The code change is tested and works on core ESP32 V.1.12.2
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by arendst** [comment]
On hold waiting for ESP32 BLE redesign by owner

**Comment by d0m1n1qu3** [comment]
this was my first merge attempt, so sorry if i made some misstakes .. my checks are fine so should be a good first merge ^^

File: tasmota/xsns_62_MI_ESP32.ino [hunk 1]
```diff
@@ -1209,6 +1214,7 @@ bool MI32Cmd(void) {
 
 const char HTTP_MI32[] PROGMEM = "{s}MI ESP32 {m}%u%s / %u{e}";
 const char HTTP_MI32_SERIAL[] PROGMEM = "{s}%s %s{m}%02x:%02x:%02x:%02x:%02x:%02x%{e}";
+const char HTTP_RSSI[] PROGMEM = "{s}%s" " RSSI" "{m}%d dBm{e}";
```
Comment (Staars) [inline tasmota/xsns_62_MI_ESP32.ino:1457]: You can use D_RSSI instead.

Comment (d0m1n1qu3) [reply tasmota/xsns_62_MI_ESP32.ino]: thx for this hint :-)

**Comment by d0m1n1qu3** [comment]
so all fixed and works fine since some hours

**Comment by Staars** [comment]
@d0m1n1qu3  Just for illustration purposes, why I think: "More is not always better":
screen

The RSSI-value is definitely not a nonsense-feature, but I hope you understand, why I try to keep the things small and lean.

BTW, please give me a few days for further work on the driver, as now seems to be a good moment to switch to more direct bridge-mode (incoming BLE goes directly to MQTT).

Thanks

**Comment by d0m1n1qu3** [comment]
under stand you in terms of gui overflow .. :-)
may be i made an other PR with some nice GUI tweaks to save room? ;-)

**Comment by Staars** [comment]
No, this is only a reminder. Please no further PR's for the moment.
First I have to remove some non-sense code of this PR. Did you really ever receive no RSSI? Makes no sense to me.

The GUI is not the real problem, but more the TELE-message:
`19:38:38 MQT: stat/tasmota_26E728/STATUS8 = {"StatusSNS":{"Time":"2020-07-14T19:38:38","LYWSD03-2ac8b3":{"RSSI":-63,"Temperature":24.7,"Humidity":47.5,"DewPoint":12.8,"Battery":100},"CGD1-508af6":{"RSSI":-64,"Temperature":24.5,"Humidity":43.6,"DewPoint":11.3},"MHOC401-a75906":{"RSSI":-65,"Temperature":24.9,"Humidity":49.1,"DewPoint":13.5},"LYWSD02-c1cc7a":{"RSSI":-71,"Temperature":24.6,"Humidity":49.0,"DewPoint":13.2},"Flora-6ab577":{"RSSI":-78,"Temperature":24.1,"Illuminance":100,"Moisture":0.000000,"Fertility":0.000000},"LYWSD03-ed815a":{"RSSI":-61,"Temperature":24.8,"Humidity":47.9,"DewPoint":13.0,"Battery":100},"MJ_HT_V1-3108be":{"RSSI":-68,"Temperature":24.6,"Humidity":46.1,"DewPoint ...`

There is not enough space anymore. Sure this would have happened with more sensors sooner or later anyway, but always keep in mind, that there are always side effects and this an example.

**Comment by d0m1n1qu3** [comment]
i got every time a rssi but the doc says that it could be empty    checking is not nonsense for me btw. 

Von meinem iPhone gesendet

**Pull request merged**